hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/snapshot/SnapshotDiffManager.java

```diff
@@ -1415,7 +1415,7 @@ long generateDiffReport(
    */
   private boolean isKeyModified(OmKeyInfo fromKey, OmKeyInfo toKey) {
     return !fromKey.isKeyInfoSame(toKey,
-        false, false, false, false)
+        false, false, false, false, false)
```

Comment:
Should key owner change show up in snapshot diffs? I guess so (should be `true` here)? @hemantk-12 @swamirishi

Reply:
Makes sense, I see the ACL was compared, and the owner may like ACL, how do you think? @hemantk-12 @swamirishi @smengcl

Reply:
Modify to true for Key owner..